jerry-core/ecma/base/ecma-helpers.c

```diff
@@ -401,26 +401,21 @@ ecma_create_property (ecma_object_t *object_p, /**< the object */
   JERRY_ASSERT (ECMA_PROPERTY_PAIR_ITEM_COUNT == 2);
 
   jmem_cpointer_t *property_list_head_p = &object_p->property_list_or_bound_object_cp;
-  bool has_hashmap = false;
 
   if (*property_list_head_p != ECMA_NULL_POINTER)
   {
-    /* If the first entry is a hashmap, it is skipped. */
+    /* If the first entry is free (deleted), it is reused. */
     ecma_property_header_t *first_property_p = ECMA_GET_NON_NULL_POINTER (ecma_property_header_t,
                                                                           *property_list_head_p);
 
+    bool has_hashmap = false;
```

Comment:
style fix: put a newline after this declaration just like below. And you can remove the newline before it.